### ISSUE TITLE
Use `west sdk`

### ISSUE THREAD

**Issue by beriberikix** [zephyrproject-rtos/action-zephyr-setup#16]
https://github.com/zephyrproject-rtos/zephyr/pull/76283 adds a west command that does the fetching of the SDK. It is much different than downloading the shell script and running it but it's a little bit cleaner.

**Comment by henrikbrixandersen** [comment]
Using `west sdk` would impose a minimum supported Zephyr version.

**Comment by fabiobaltieri** [comment]
> Using `west sdk` would impose a minimum supported Zephyr version.

Good point, and it's not in the LTS so that's  a bit of a problem. May be a change big enough to justify a `v2` tag. Either way,  I guess we have to wait for at least 4.0 before it make sense to look into it.
